packages/runtime/container-runtime/src/dataStores.ts

```diff
@@ -476,41 +527,24 @@ export class DataStores implements IDisposable {
 			!this.contexts.has(id),
 			0x570 /* Inconsistent state! GC says the data store is deleted, but the data store is not deleted from the runtime. */,
 		);
-		sendGCUnexpectedUsageEvent(
```

Comment:
Is this thing used anymore?

Reply:
Yeah, it is. I am not sure the flags it logs make sense anymore. We should take a look and remove its not needed anymore.

Reply:
Added it to our Follow Ups list